### ISSUE TITLE
[Queue] Switch from kue

### ISSUE THREAD

**Issue by rohit-gohri** [smartprix/sm-utils#10]
Kue is no longer being maintained, https://twitter.com/automattic/status/1019073146446602240

Possible alternatives: https://github.com/OptimalBits/bull

**Comment by smartprix** [comment]
Yes, bull is a good choice. Switching to that would be a good idea.

**Comment by rohit-smpx** [comment]
kue is deprecated (https://github.com/Automattic/kue/pull/1231). Need to switch soon
